src/train.py

```diff
@@ -8,17 +8,25 @@
 from sklearn.metrics import confusion_matrix
 import matplotlib.pyplot as plt
 import pantry
+from fridge import model_saver as saver
 
 np.set_printoptions(threshold=np.inf)  # Print full confusion matrix.
 FLAGS = flags.FLAGS
+
 flags.DEFINE_string(
     "pastry", None,
-    "Pastry: indicates network, optimizer and number of epochs")
+    "Pastry: indicates discriminator network, optimizer and number of epochs")
```

Comment:
Eventually we should move the number of epochs out as a separate flag. I'll do that in a subsequent PR.